fix: LEAP-913: LEAP-926: LEAP-1007: Fix security vulnerabilities

juliosgarbi opened the pull request:
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend


#### What does this fix?
We are updating some packages to fix some security vulnerabilities

juliosgarbi committed:
fix: LEAP-913: LEAP-926: LEAP-1007: Fix security vulnerabilities

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9213646340

juliosgarbi committed:
add express package in the resolutions

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9213714410

hlomzik commented on web/package.json:
```diff
@@ -213,7 +212,8 @@
     "@babel/traverse": "^7.24.1",
-    "ip": ">=2.0.1"
+    "ip": ">=2.0.1",
+    "express": ">=2.0.1"
```
order is broken here and for some packages above. this might be auto-resolved by yarn in next run producing unclear change, so let's reorder it here and now

juliosgarbi committed:
reordering resolutions

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9225609365
